src/lib/modules/Literature/LiteratureUrls.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Overridable from 'react-overridable';
+import { Icon } from 'semantic-ui-react';
+
+class LiteratureUrls extends Component {
+  render() {
+    const { urlArray, separator } = this.props;
+    return (
+      <Overridable id="LiteratureUrls.layout">
+        <>
+          {urlArray.map((url, index) => {
+            const separatorCmp = index < urlArray.length - 1 ? separator : null;
+            return (
+              <>
+                <Icon name="linkify" />
+                <a href={url.value} key={url.value}>
+                  {url.description ? url.description : url.value}
+                </a>
+                {separatorCmp}
+              </>
+            );
+          })}
+        </>
+      </Overridable>
+    );
+  }
+}
+
+LiteratureUrls.propTypes = {
+  urlArray: PropTypes.array.isRequired,
+  separator: PropTypes.string,
```

Comment:
String or component as a separator? Isn't it maybe better to display a list?

Reply:
I tried with a list, it does not look good IMO, in terms of leaving a lot of blank space when there is many links and each has a short description